resultsdbupdater/utils.py

```diff
@@ -236,7 +236,10 @@ def handle_ci_umb(msg):
         system = system[0] if system else {}
 
     if item_type == 'productmd-compose':
-        item = msg_body['artifact']['compose_id']
+        architecture = system['architecture']
+        variant = system.get('variant')
+        compose_id = msg_body['artifact']['compose_id']
+        item = '{0}/{1}/{2}'.format(compose_id, variant or '', architecture)
```

Comment:
Instead of the `or ''` you could just change line 240 to `variant = system.get('variant', '')`. Although, it likely should be something like `unknown` or else there'll be two consecutive slashes in the item name which seems odd.

Reply:
If variant is `None` it will be dropped out from the result_data dict below.

I changed the empty variant to `unknown`.